### PR TITLE
Create a Post

### PR DESCRIPTION
**Pull request by FORDBA**
# Description
Please include a summary of the change and which issue is fixed / implemented. Please also include relevant motivation and context. List any dependencies that are required for this change (e.g., do you need to run an npm install to test?).
Fixes # (issue) # (issue2) ... (these should be the numbers of issues we have created / on the project board for this project)
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# How Has This Been Tested?
Please describe step-by-step the process that a reviewer can follow to adequately test this PR. 
- [ ] This requires the server side mgj-add-post branch to be checked out unless it has been merged into main
- [ ] Create a post and check to see that it appears in the post list
- [ ] Check database to ensure all attributes are correct on the new post.
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/components/posts/postForm.js [hunk 1]
```diff
@@ -32,22 +31,19 @@ export const PostForm = (props) => {
     const populateFormValues = post => {
       titleRef.current.value = post.title
       categoryRef.current.value = post.category_id
-      imageRef.current.value = post.image
+      imageRef.current.value = post.image_url
       contentRef.current.value = post.content
 
       // HTML date inputs take their value in the format YYYY-MM-DD
```
Comment (skratz17) [inline src/components/posts/postForm.js:37]: This comment no longer refers to anything... can we get rid of it?

Comment (FORDBA) [reply src/components/posts/postForm.js]: This is removed

**Review by skratz17 — APPROVED** [review]
All seems correct to me.

**Review by gavinschriver — APPROVED** [review]
sheeeeee workin

**Pull request merged**